src/modules/entities/services/reducers/api.js

```diff
@@ -1,8 +1,8 @@
 import { combineReducers } from 'redux';
 import { ApiKeys } from '../../../../constants';
-import { apiReducer as filters } from '../../modules/filters';
 
-import { apiReducer as reports } from '../../modules/reports';
```

Comment:
When you are in the same module (here - modules/entities) its not bad when you do it this way :) both ways are cool